### ISSUE TITLE
Website accessibility is insufficient

### ISSUE THREAD

**Issue by csaluski** [mpctnano-dev/Radiant_Alpha#2]
I ran the live site through an accessibility checker, and it says there are several elements that are outside of WCAG requirements. https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fradiant.nau.edu%2F&flag=us#

NAU requires websites to be accessible, see [Web Accessibility](https://in.nau.edu/Accessibility/Web-Accessibility/) and [Resources](https://in.nau.edu/Accessibility/Resources/).
